### PR TITLE
Migrate to poetry

### PR DESCRIPTION
**Pull request by tetienne**
- Migrate to Poetry

- Add pre-commit with some linters:

  - flake8
  - pylintrc
  - mypy
  - isort
  - black

- Add Pycharm configuration to ease the development setup

- Update the README.md to explain how to contribute to this project

### CODE REVIEW

**Comment by tetienne** [comment]
For the moment, the linters output some errors. I will have to fix them.

**Comment by tetienne** [comment]
@iMicknl https://github.com/iMicknl/python-tahoma-api/pull/2/files#diff-b30393141377169ab3c3dc6750a3f3c0L119 I cannot find any Command class. Did you forget it or it's a mistake?

**Comment by vlebourl** [comment]
it's in another branch: [add_command_implementation](https://github.com/iMicknl/python-tahoma-api/tree/add_command_implementation)

**Comment by tetienne** [comment]
@vlebourl Thx. I will have to include it otherwise the check will not pass. I've clean all the remarks except this one. 
I will move also the test.py within an examples folder.

**Comment by vlebourl** [comment]
Can you add a `setup.cfg` for the precommit config? Or is it setup somewhere else?

**Comment by tetienne** [comment]
@vlebourl The pre-commit configuration has its own file. According to its author it's because this framework is not Python specific, and so nodejs user for instance does not care about setup.cfg.

**Comment by tetienne** [comment]
@iMicknl I've commented the method using the Command type. I let you from the master on this branch merged.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
+
 An updated and async version of the original [tahoma-api](https://github.com/philklei/tahoma-api) by [@philklei](https://github.com/philklei). The aim of this wrapper is to offer an easy to consume Python wrapper for the internal API's used by tahomalink.com.
 
 Somfy TaHoma has an official API, which can be consumed via the [Somfy-open-api](https://github.com/tetienne/somfy-open-api). Unfortunately only a few device classes are supported via the official API, thus the need for this wrapper.
 
 This package is written for the Home Assistant [ha-tahoma](https://github.com/iMicknl/ha-tahoma) integration, but could be used by any Python project interacting with Somfy TaHoma devices.
+
+# Development
+
+## Installation
```
Comment (vlebourl) [inline README.md:16]: Dev on VSCode follows this method?

Comment (tetienne) [reply README.md:16]: Yes the installation part is not linked to any IDE. That's the workflow I usualy follow. You should be able to start from  scratch in 5 min only.

**Comment by tetienne** [comment]
> I've been looking at poetry for a while, I guess this is a good way to start dipping into it!

You have to know a little setup.py to enjoy the real power of Poetry :D

File: README.md (hunk 1 above)
Comment (vlebourl) [reply README.md:16]: I use PyCharm any ways, so method 2 is good enough for me!

**Pull request merged**